topics/ecology/tutorials/champs-blocs/tutorial.md

```diff
@@ -0,0 +1,200 @@
+> > 2. Browse to your Galaxy instance
+> > 3. Log in or register
+> >
+> > ![Screenshot of Galaxy interface with the register or login button highlighted](../../images/champbloc/galaxy_homepage.png "Galaxy homepage")
+> >
+> > This is an image of Galaxy Ecology, located at [usegalaxy.eu](https://ecology.usegalaxy.eu/)
+> >
+> {: .hands_on}
+>
+> The Galaxy homepage is divided into three panels:
+> * Tools on the left
+> * Viewing panel in the middle
+> * History of analysis and files on the right
+>
+> ![Screenshot of the Galaxy interface, the tools panel is on the left, the main panel is in the center, and the history is on the right.](../../images/champbloc/galaxy_interface.png "Galaxy interface explanation")
+>
+> The first time you use Galaxy, there will be no files in your history panel.
+{: .details}
+
+Now let's focus on our workflow on boulder field ecological state
+
+![Workflow for boulder field ecological state analysis](../../images/champbloc/workflow.png "Workflow")
+
+# Get data
+
+{% include _includes/cyoa-choices.html option1="Yes" option2="No" default="Yes" text="Are your ESTAMP data ready ?" %}
+
+<div class="No" markdown="1">
+> 1. Download your data on ESTAMP [estamp.afbiodiversite.fr](https://estamp.afbiodiversite.fr/donnees). You will get a zip folder.
+>
+> 2. Unzip your folder. In the folder three files .csv will interest us :
+> - champbloc_ivr.csv
+> - champbloc_qecb.csv
+> - ficheterrain.csv
+</div>
+
+<div class="Yes" markdown="1">
+</div>
+
+> <hands-on-title>Data upload</hands-on-title>
+> 1. Import the data into Galaxy
+> > * Open the Galaxy Upload Manager {% icon galaxy-upload %}
+> > * Select **Choose local files**
+> > ![Choose local files](../../images/champbloc/upload.png "upload your data")
+> > * Browse in your computer and get your ESTAMP files (select all three of them : champbloc_ivr.csv, champbloc_qecb.csv and ficheterrain.csv)
+> >
+> > * Press **Start**
+> > ![Start charging your data](../../images/champbloc/start.png "Start")
+> > * Press **Close**
+> > ![Close upload your data](../../images/champbloc/close.png "Close")
+> > You need to wait for the data to become green on the right panel
+> > ![Data ready](../../images/champbloc/after_before.png "Data ready")
+> >
+> > <tip-title>Tip: Create a new history</tip-title>
+> > Create a new history for this tutorial and give it a name (example: “Champs blocs indicators”) for you to find it again later if needed.
+> > {% snippet faqs/galaxy/histories_create_new.md box_type="none" %}
+> {: .tip}
+{: .hands_on}
+
+
+# Computing the Visual Boulder turning Indicator for each site
+
+Based on the proportion of "turned" and "not turned" blocs this indicator vary from 0 to 5 and can be quickly used.
+
+> <tip-title>How to get your tool ?</tip-title>
+>
+> > <hands-on-title>Use a tool</hands-on-title>
+> > 1.  click on for example the {% tool [IVR](toolshed.g2.bx.psu.edu/repos/ecology/cb_ivr) %} tool or type **ivr** in the tools panel search box (top left)
```

Comment:
Here it is missing a tool ID and tool version, this is just a toolshed repository owned by ecology, named cb_ivr, it doesn't list the specific tool inside that repository.

If you can get the full tool url (see the [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#how-to-find-these-ids)) that will fix it.

If you're using IUC standard tool repository setups, then it's probably

toolshed.g2.bx.psu.edu/repos/ecology/cb_ivr/cb_ivr/<some-version>, the `cb_ivr` duplication due to every tool getting placed into a separate tool repository on the toolshed